relay-dynamic-config/src/feature.rs

```diff
@@ -95,6 +95,11 @@ pub enum Feature {
     /// Serialized as `organizations:ourlogs-ingestion`.
     #[serde(rename = "organizations:ourlogs-ingestion")]
     OurLogsIngestion,
+    /// Enable trace metric ingestion for our trace metric product.
+    ///
+    /// Serialized as `organizations:tracemetrics-ingestion`.
+    #[serde(rename = "organizations:tracemetrics-ingestion")]
```

Comment:
nit: For consistency with the camel case name. Not a strong opinion.

```suggestion
    #[serde(rename = "organizations:trace-metrics-ingestion")]
```

Reply:
The backend is already merged, but you're right 😅